marimo/_output/data/data.py

```diff
@@ -33,3 +43,45 @@ def image(data: bytes, ext: str = "png") -> VirtualFile:
     item = VirtualFileLifecycleItem(ext=ext, buffer=data)
     get_context().cell_lifecycle_registry.add(item)
     return item.virtual_file
+
+
+def any_data(data: Union[str, bytes, io.BytesIO], ext: str) -> VirtualFile:
+    """Create a virtual file from any data.
+    It can be a string, bytes, or a file-like object.
+    For external URLs, these are passed through as-is.
+
+    **Args.**
+
+    - data: Data in bytes
+    - ext: File extension
+
+    **Returns.**
+
+    A `VirtualFile` object.
+    """
+    if data is None:
+        return EMPTY_VIRTUAL_FILE
+
+    if is_data_empty(data):
+        return EMPTY_VIRTUAL_FILE
+
+    # Base64 encoded data
+    if isinstance(data, str) and data.startswith("data:"):
+        buffer = data.split(",")[1]
+        item = VirtualFileLifecycleItem(ext=ext, buffer=buffer)
```

Comment:
ok, ill convert or cast

Reply:
I don't think casting will work, the buffer needs to be `bytes` (or otherwise implement the buffer protocol)  because it gets stored in a `memoryview` (https://docs.python.org/3/library/stdtypes.html?#memoryview). If it doesn't raise an error it's probably undefined behavior